tests/Integration/Fixtures/Techproducts/gb18030-example.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="GB18030"?>
```

Comment:
I assume, the PR isn't finished yet, right?
Posting the test data via solarium would be a great addition to the integration tests :-)

Reply:
My bad. I thought I had those changes stashed away. Wasn't meant to be added just yet.

I was working on importing them as suggested in #751 by yourself. Didn't get to finish it yet, work (and — it shows — sleep) schedule have been impacted by current world affairs.

I'll remove them for now, and commit them with the PR they were meant for.